examples/booleans/booleans.bal

```diff
@@ -1,16 +1,9 @@
 import ballerina/io;
 
-boolean flag = true;
-
-// Here's a conditional expression. Uses C syntax.
-int n = flag ? 1 : 2;
-
 public function main() {
-    // Parentheses are optional in conditions
-    // but curly braces are required in `if/else` and other compound statements.
-    if flag {
-        io:println(1);
-    } else {
-        io:println(2);
-    }
-}
+    boolean flag = true;
+    io:println(flag);
+
+    // following example will output `false`
+    io:println(3 < 2);
```

Comment:
Break this into two variable definitions and compare. `3 < 2` Always generates a warning.